src/@next/Icon/icons/ri-hour-glass-line.svg

```diff
@@ -0,0 +1,6 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+  <path fill-rule="evenodd" clip-rule="evenodd"
```

Comment:
If I delete `fill-rule="evenodd" clip-rule="evenodd"` it will not render the icon correctly

![image](https://user-images.githubusercontent.com/52077891/227168191-7026d50d-12d9-4918-867f-fefc29e74659.png)